taichi/backends/cuda/cuda_device.h

```diff
@@ -0,0 +1,117 @@
+#pragma once
+#include <vector>
+#include <set>
+
+#include "taichi/common/core.h"
+#include "taichi/backends/cuda/cuda_driver.h"
+#include "taichi/backends/cuda/cuda_context.h"
+#include "taichi/backends/device.h"
+
+TLANG_NAMESPACE_BEGIN
```

Comment:
nit: let's stop using macros

Reply:
Let's have @taichi-gardener submit a PR that removes all of them?

Reply:
Sounds good!